scripts/build/fiwarelab/build-fiwarelab-image.sh

```diff
@@ -138,5 +150,209 @@ sudo iptables -I INPUT 1 -p tcp -m state --state NEW -m tcp --dport 1026 -j ACCE
+sudo yum -y install gcc
+wget http://download.redis.io/releases/redis-2.6.17.tar.gz
+tar xfvz redis-2.6.17.tar.gz
+rm redis-2.6.17.tar.gz
+cd redis-2.6.17/deps
+make hiredis jemalloc linenoise lua
+cd ..
+make
+sudo make install
+sudo mkdir /etc/redis
+sudo mkdir /var/redis
+REDIS_CONFIG=$(mktemp)
+cp redis.conf ${REDIS_CONFIG}
+sed -i "/daemonize no/c\daemonize yes" ${REDIS_CONFIG}
+sed -i "/dir .\//c\dir /var/redis" ${REDIS_CONFIG}
+sed -i "/logfile ""/c\logfile /var/log/redis.log" ${REDIS_CONFIG}
+sudo mv $REDIS_CONFIG /etc/redis/redis.conf
+cd ..
+rm -r redis-2.6.17
+
+# Redis init script from https://gist.github.com/paulrosania/257849
+REDIS_INIT=$(mktemp)
+cat > $REDIS_INIT <<'EOF'
+#!/bin/sh
+#
+# redis - this script starts and stops the redis-server daemon
+#
+# chkconfig:   - 85 15 
+# description:  Redis is a persistent key-value database
+# processname:  redis-server
+# config:      /etc/redis/redis.conf
+# config:      /etc/sysconfig/redis
+# pidfile:     /var/run/redis.pid
+
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+# Source networking configuration.
+. /etc/sysconfig/network
+
+# Check that networking is up.
+[ "$NETWORKING" = "no" ] && exit 0
+
+redis="/usr/local/bin/redis-server"
+prog=$(basename $redis)
+
+REDIS_CONF_FILE="/etc/redis/redis.conf"
+
+[ -f /etc/sysconfig/redis ] && . /etc/sysconfig/redis
+
+lockfile=/var/lock/subsys/redis
+
+start() {
+    [ -x $redis ] || exit 5
+    [ -f $REDIS_CONF_FILE ] || exit 6
+    echo -n $"Starting $prog: "
+    daemon $redis $REDIS_CONF_FILE
+    retval=$?
+    echo
+    [ $retval -eq 0 ] && touch $lockfile
+    return $retval
+}
+
+stop() {
+    echo -n $"Stopping $prog: "
+    killproc $prog -QUIT
+    retval=$?
+    echo
+    [ $retval -eq 0 ] && rm -f $lockfile
+    return $retval
+}
+
+restart() {
+    stop
+    start
+}
+
+reload() {
+    echo -n $"Reloading $prog: "
+    killproc $redis -HUP
+    RETVAL=$?
+    echo
+}
+
+force_reload() {
+    restart
+}
+
+rh_status() {
+    status $prog
+}
+
+rh_status_q() {
+    rh_status >/dev/null 2>&1
+}
+
+case "$1" in
+    start)
+        rh_status_q && exit 0
+        $1
+        ;;
+    stop)
+        rh_status_q || exit 0
+        $1
+        ;;
+    restart|configtest)
+        $1
+        ;;
+    reload)
+        rh_status_q || exit 7
+        $1
+        ;;
+    force-reload)
+        force_reload
+        ;;
+    status)
+        rh_status
+        ;;
+    condrestart|try-restart)
+        rh_status_q || exit 0
+	    ;;
+    *)
+        echo $"Usage: $0 {start|stop|status|restart|condrestart|try-restart|reload|force-reload}"
+        exit 2
+esac
+EOF
+sudo mv $REDIS_INIT /etc/init.d/redis
+sudo chmod a+x /etc/init.d/redis
+
 # Install Rush
-# TBD
+wget https://github.com/telefonicaid/Rush/archive/1.8.3.tar.gz
+tar xfvz 1.8.3.tar.gz
+sudo mkdir -p /opt
+sudo mv Rush-1.8.3/ /opt/Rush
+rm -rf 1.8.3.tar.gz
+
+RUSH_INIT=$(mktemp)
+cat > $RUSH_INIT <<'EOF'
+#!/bin/bash
```

Comment:
I understand that the Rush init script is not available in the Rush repository, thus I'm using a here doc based on the init script currently in orion-pab-image-R4.2.